src/tools/ci/docker/linux_fedora_rawhide

```diff
@@ -0,0 +1,49 @@
+FROM fedora:rawhide
+RUN \
+  yum -y update && \
```

Comment:
I think these days dnf is the recommended thing to install packages on red had distros?

Reply:
Done.